zerver/views/auth.py

```diff
@@ -501,6 +501,9 @@ def api_dev_fetch_api_key(request, username=REQ()):
     if return_data.get("inactive_user"):
         return json_error(_("Your account has been disabled."),
                           data={"reason": "user disable"}, status=403)
+    if user_profile is None:
+        return json_error(_("This user is not registered."),
+                          data={"reason": "unregistered"}, status=403)
```

Comment:
@YagoGG can you add a test for this?  That's why Travis CI is failing -- it requires 100% test coverage for this file

I think the "invalid subdomain" code path is probably how one gets here.